### PR TITLE
Update phone number field to use international format

### PR DESCRIPTION
**Pull request by sangkips**
This option allows users to register phone number details with there own country specific prefix

### CODE REVIEW

**Comment by JimmyTron** [comment]
We could have first merged the fix to the bug then embark on this.

**Comment by JimmyTron** [comment]
> We could have first merged the fix to the bug then embark on this. 


You could resolve the conflict first

File: README.md [hunk 1]
```diff
@@ -26,19 +26,57 @@ For a more detailed overview of the project, read through the [CMS Backend wiki]
...
+]
+```
+
+- Model field section add the following:
+
+```
+from phonenumber_field.modelfields import PhoneNumberField
+
+phone_number = PhoneNumberField(blank=True)
+```
```
Comment (JimmyTron) [inline README.md:82]: This has actually answered my questions on what was really happening. 👍 Nice

File: accounts/models.py [hunk 2]
```diff
@@ -10,23 +12,24 @@
...
 class User(AbstractUser):
     email = models.EmailField(unique=True, max_length=50)
-    phone_number = models.CharField(max_length=30, blank=True)
+    phone_number = PhoneNumberField(blank=True, help_text='Contact phone number', null=True)
     verification_code = models.CharField(max_length=100, blank=True)
 
+    
     USERNAME_FIELD = 'email'
-    REQUIRED_FIELDS = ['first_name', 'last_name', 'username']
+    REQUIRED_FIELDS = ['first_name', 'last_name', 'username', 'phone_number'] # add phone number as a requirement while signing up
```
Comment (JimmyTron) [inline accounts/models.py:20]: This will actually force us to change the field property from ```null=true``` to false. Or what?

**Review by JimmyTron — APPROVED** [review]
Haven't spotted anything odd.

File: requirements.txt [hunk 3]
```diff
@@ -4,3 +4,5 @@ djangorestframework==3.14.0
 pytz==2022.6
 sqlparse==0.4.3
 tzdata==2022.6
+django-phonenumber-field==7.0.0 # enables localization of phone number based on couuntry code
```
Comment (JimmyTron) [inline requirements.txt:7]:
```suggestion
django-phonenumber-field==7.0.0 # enables localization of phone number based on country code

django-phonenumbers
```

**Review by JimmyTron — COMMENTED** [review]
add the ```django-phonenumbers``` module to the requirements file

**Review by wanjirumurira — APPROVED** [review]

**Pull request merged**